packages/react-scripts/config/webpack.config.js

```diff
@@ -88,7 +88,10 @@ module.exports = function (webpackEnv) {
   // common function to get style loaders
   const getStyleLoaders = (cssOptions, preProcessor) => {
     const loaders = [
-      isEnvDevelopment && require.resolve('style-loader'),
+      isEnvDevelopment && {
+        loader: require.resolve('style-loader'),
+        options: {},
```

Comment:
Would it be better to leave this undefined?

Reply:
43da7d3